solutions/USACO dec18-bucketlist.mdx

```diff
@@ -41,3 +41,48 @@ file_out.close()
 </PySection>
 
 </LanguageSection>
+
+<LanguageSection>
+
+<CPPSection>
+
+```cpp
+#include <iostream>
+#include <algorithm>  
+#include <cstdio>
+using namespace std;
+int event[1001]; // declaring the array globaly to intialize all the elements to 0
```

Comment:
```suggestion
int event[1001]; // declaring the array globally to initialize all the elements to 0
```

Reply:
Also, can we have a better name than `event` :P (or alternatively explain what event is)